bin/merge.sh

```diff
@@ -176,7 +172,7 @@ main() {
       usage
       ;;
   esac
+
```

Comment:
I also added newlines to match the dominating style convention in the rest of this file.